### ISSUE TITLE
Dart support

### ISSUE THREAD

**Issue by bystepii** [kaitai-io/kaitai_struct#1009]
How difficult it would be to add Dart support? And if I wanted to implement it myself, where should I start?

**Comment by GreyCat** [comment]
http://doc.kaitai.io/new_language.html gives an overview of the process, please take a look.

**Comment by tvladyslav** [comment]
@bystepii Are you working on this?

**Comment by bystepii** [comment]
@tvladyslav No, currently I have no time to add Dart, and the whole process is a bit complex and long.
